### DISCUSSION TITLE
Yunohost: error during script installation

### DISCUSSION THREAD

**Discussion by matkes** [tteck/Proxmox#3788] [Help Needed]
I'm getting a similar error message (as in #1436 ) during installation:
✓ Using local for Template Storage.
✓ Using local-lvm for Container Storage.
✓ Updated LXC Template List
✓ LXC Container 108 was successfully created.
✓ Started LXC Container
✓ Set up Container OS
✓ Network Connected: 10.0.0.254
✓ IPv4 Internet Connected
✗ IPv6 Internet Not Connected
✓ DNS Resolved github.com to 140.82.121.3
✓ Updated Container OS
✓ Installed Dependencies
/ Installing YunoHost (Patience)
[ERROR] in line 23: exit code 0: while executing command "$@" > /dev/null 2>&1
The silent function has suppressed the error, run the script with verbose mode enabled, which will provide more detailed output.

Other LXC CTs from your library do install on my end with no errors.

**Comment by tteck** [comment]
🛑 Failure to comply with the following guideline.
> - When encountering the error message [ERROR] in line 23: exit code *: while executing command "$@" > /dev/null 2>&1, make sure to run the script in verbose mode to accurately determine the underlying issue.

**Comment by matkes** [comment]
One day later, another try, and the installation was successful.
